Apps/WebClient/src/ClientApp/src/views/landing.vue

```diff
@@ -283,7 +285,7 @@
                         </b-col>
                     </b-row>
                     <b-row class="py-3">
-                        <b-col>
+                        <div class="col-12 d-flex flex-column flex-sm-row">
```

Comment:
no success with the grid system?

Reply:
no, as I have mentioned above "the space between 2 buttons is not easy to control without complex code/tags".
What is your concerns about bootstrap's d-flex and flex-column here?

Reply:
Having two columns stack when the container is not too big is one of the big features of the bootstrap grid system. The concern is that there are a large number of examples on the docs that we would not be leveraging for unnecessary custom code
https://getbootstrap.com/docs/4.0/layout/grid/